boot/storage.txt

```diff
@@ -0,0 +1,285 @@
+    |                  |   MBR) and cannot be placed elsewhere.                |
+    |                  |                                                       |
+    +------------------+-------------------------------------------------------+
+
+    * Due to using 32-bit identifiers, DOS partitioning tables cannot handle 
+      disks that are >2 TBs in size. Also, unless a extended partition is 
+      created, DOS disklables supports a maximum of four partitions.
+    * Using GPT on a BIOS-based computer works, but then one cannot dual-boot 
+      with a Microsoft Windows operating system. The reason is that Microsoft 
+      Windows will boot in UEFI mode if it detects a GPT partition label. Also 
+      note that some buggy (old) motherboard firmware configured to boot in 
+      BIOS/CSM/legacy mode might also have problems with booting from GPT  
+      labeled disks.
+
+
+    [1.2] Tools
+    ____________________________________________________________________________
+
+    Depending on which Live CD or distribution (e.g. Arch or Gentoo) KISS Linux
+    is installed from, you may find one or several of the following tools at 
+    your disposal:
+
+    - gdisk
+    - parted
+    - fdisk
+    
+    
+[2.0] Partitioning Schemas
+________________________________________________________________________________
+
+Before creating a partition or multiple partitions, there are several things to
+take into consideration:
+
+    [2.1] Partition Number, Format and Size
+    ____________________________________________________________________________
+    
+    The number of partitions required is highly variable and dependent on the
+    environment.  The following table provides examples of common partition
+    schemas, formats and recommended sizes:
+    
+    +-------------------------------+------------------------------------------+
+    |    Schema Description         |   Partition (Type,      Format, Size)    |
+    +-------------------------------+------------------------------------------+
+    |                               |                                          |
+    |    GPT with UEFI Support      |   /dev/sda1 (BIOS BOOT, EXT4,   128MB)   |
+    |    with physical swap         |   /dev/sda2 (UEFI ESP,  FAT32,  33M)     |
+    |                               |   /dev/sda3 (SWAP,      SWAP,   >512MB)  |
+    |                               |   /dev/sda4 (ROOTFS,    EXT4,   *)       |
+    |                               |                                          |
+    |    GPT with UEFI Support      |   /dev/sda1 (BIOS BOOT, EXT4,   128MB)   |
+    |    wihout physical swap       |   /dev/sda2 (UEFI ESP,  FAT32,  33M)     |
+    |                               |   /dev/sda3 (ROOTFS,    EXT4,   *)       |
+    |                               |                                          |
+    +-------------------------------+------------------------------------------+
```

Comment:
Because of [2.3], I think these examples are a bit lacking. There are three major use cases: strict UEFI, strict legacy, and hybrid (GPT + BIOS). Because EUFI is 'the future' (and has been for the last... ten years... smh), this should be first, then a strict BIOS setup, and then the (perhaps unusual? It only seems necessary for bad UEFI implementations and those have gotten better and better since 2009) GPT+BIOS configuration. 
The distinction between 'with/without swap' can be replaced and instead we could just lump it into one of the examples.

Reply:
how do we feel about suggesting a separate home partition?

Reply:
This section is kind of a mess at the moment. Will try to clarify shortly (including mention of a /home partition).